config/warren_traction_dev.yml

```diff
@@ -0,0 +1,15 @@
+---
+MultiLimsWarehouseConsumer:
+  desc: Configuration compatible with traction development settings
+  queue:
+    name: psd.traction.to-warehouse
+    options:
+      :durable: true
+    bindings:
+    - exchange:
+        name: bunny.examples.exchange
+        options:
+          :type: topic
+          :durable: true
+      options:
+        :routing_key: "#"
```

Comment:
The Readme implied that traction is using a different incompatible configuration. However, the suggestion to modify the config files that was there previously is likely to lead to accidental commits, and possibly changing production config. Fortunately we can specify a custom configuration file, and here I've set one up that follows the instructions. This avoids the need to modify files.

Reply:
Deleted lines 82-86 in the readme suggested that the Tesla guys ran into issues when setting up the warehouse with traction, because the suggested setups didn't match. Their suggested solution was modifying the config files, but this kind of thing just results in changes getting accidentally committed. So I just set up a yaml file that mimicked the setup in the traction example file, to help people get up to speed. The Readme details how you can run with this config instead.
I'll add *.local.yml to the git-ignore as well, just in case anyone has their own configs they like to play with.